sites/dotnet-guide/contents/tutorials/resharper-to-rider/index.md

```diff
@@ -0,0 +1,21 @@
+---
+date: 2021-01-28
+title: Moving from Visual Studio + Resharper to JetBrains Rider.
+technologies: [.net]
+topics: [ide]
+products: [rider,resharper]
+author: khalidabuhakmeh
+subtitle: A jumpstart guide for Visual Studio + Resharper users.
+thumbnail: ./thumbnail.png
+tutorialItems: 
+  - /tutorials/resharper-to-rider/getting-started-for-resharper-users/
+  - /tutorials/resharper-to-rider/rider-debugger-for-resharper-users/
+  - /tutorials/resharper-to-rider/working-with-databases/
+  - /tutorials/resharper-to-rider/unit-testing-for-resharper-users/
+  - /tutorials/resharper-to-rider/visual-studio-rider-toolbar/
+  - /tutorials/resharper-to-rider/nuget-package-management/
+  - /tutorials/resharper-to-rider/plugins-for-developers/
+  - /tutorials/resharper-to-rider/working-with-internet-information-services-iis/
+---
+
+Many of the same features we've all grown to love in [ReSharper](https://jetbrains.com/resharper) help power [JetBrains Rider's](https://jetbrains.com/rider) development experience. While most functionality is still present, it may take us a little effort to learn where everything is within a brand new development environment. In this guide, we'll look at some everyday functionality folks might find helpful when migrating from the combination of Visual Studio and ReSharper.
```

Comment:
```suggestion
Many of the same features we've all grown to love in [ReSharper](https://www.jetbrains.com/resharper/) help power [JetBrains Rider's](https://www.jetbrains.com/rider/) development experience. While most functionality is still present, it may take us a little effort to learn where everything is within a brand new development environment.

In this guide, we'll look at some everyday functionality folks might find helpful when migrating from Visual Studio, with or without ReSharper.
```